cli/command/service/logs.go

```diff
@@ -44,6 +50,8 @@ func newLogsCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVar(&opts.noResolve, "no-resolve", false, "Do not map IDs to Names")
+	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Do not truncate output")
+	flags.BoolVar(&opts.noIDs, "no-ids", false, "Do not include task IDs")
```

Comment:
Perhaps this should be `no-task-ids` in case the format includes other types of IDs in the future?

Or is it better to just include those IDs under this option as well if/when that happens?